app/controllers/api_controller.rb

```diff
@@ -86,6 +86,7 @@ def set_access_control_headers
   include_concern 'TenantQuotas'
   include_concern 'Users'
   include_concern 'Vms'
+  include_concern 'ContainerDeployments'
```

Comment:
this was alphabetized once. submitted #8943, you can move this to the top after Conditions
 @alongoldboim